### ISSUE TITLE
Question about improvement

### ISSUE THREAD

**Issue by KingOlami** [eonist/HybridCamera#3]
Great work in all but is it possible to adapt this to a Storyboard design since you made the project without a storyboard. cheers

**Comment by eonist** [comment]
@kingOlami yes. You would need to connect the storyboard views with iboutlets etc. Im going to pay attrntion to this lib in about a week, and i can Add a storyboard setup AS well. Should take aprox 30min

**Comment by KingOlami** [comment]
that is awesome. @eonist looking forward to it.

**Comment by Mitesh-Mewada** [comment]
@eonist  i implement this code in  storyboard, set UIView  added class **HybridCamView**
when i click record button and change camera mode it's not working, Any idea how to resolve this problem.

class Viewmains: UIViewController {

    @IBOutlet weak var viewCamera: HybridCamView!
    override func viewDidLoad() {
        super.viewDidLoad()
//self.initiate()
    }
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        CamUtil.assertVideoAndMicAccess(vc: self){ (success:Bool) in
            guard success else {return}
            DispatchQueue.main.async {
                self.initiate()
            }
        }
    }
    override var shouldAutorotate: Bool  {return false}/*Locks autorotate*/
    override var preferredStatusBarStyle: UIStatusBarStyle { return .lightContent }
}
extension Viewmains{
    /**
     * Common init
     */
    func initiate(){
        let hybridCamView = HybridCamView()
      //  self.view = hybridCamView
        //self.viewCamera.addSubview(hybridCamView)
        hybridCamView.camView.onPhotoCaptureComplete = onCapture
        hybridCamView.camView.onVideoCaptureComplete = {(url:URL?,error:Error?)in self.onCapture(nil,url,error)}
        self.viewCamera.addSubview(hybridCamView)

    }
    /**
     * When camera onCapture is called
     */
    private func onCapture(_ image:UIImage?,_ url:URL?,_ error:Error?) {
        let processMediaView = ProcessMediaView.init(frame: UIScreen.main.bounds)
        processMediaView.onExit = {processMediaView.deInitiate()}
        processMediaView.onShare = { (url:URL?) in if let url = url {ProcessMediaView.promptSaveFile(vc: self, url: url, onComplete: {processMediaView.deInitiate()})}}
        self.view.addSubview(processMediaView)
        if let error = error{
            ProcessMediaView.promptErrorDialog(vc: self, error: error, onComplete: {processMediaView.deInitiate()});return
        }else {
            
            processMediaView.present(image: image, url: url)
        }
    }
}

**Comment by eonist** [comment]
@Mitesh-Mewada Could be related to: https://github.com/eonist/HybridCamera/issues/2 Im sure its a simple fix. Unfortunately I won't have time to test this bug until the weekend. I will add Carthage to the project and I can try to fix the bug. 👍

**Comment by KingOlami** [comment]
Good work champ
